File: examples/pascal/constructs/scope_leak.pas
// Ported from examples/lumen/constructs/scope_leak.lm by scripts/port_examples.py; edit the Lumen original, not this file.
var y: integer;

begin
    y := 100;
    writeln(y);
    if true then begin
        y := 50;
        writeln(y);
    end;
    writeln(y);
end.

File: examples/c/constructs/scope_leak.c
// Ported from examples/lumen/constructs/scope_leak.lm by scripts/port_examples.py; edit the Lumen original, not this file.
int main(void) {
    long y = 100;
    printf("%ld\n", y);
    if (true) {
        y = 50;
        printf("%ld\n", y);
    }
    printf("%ld\n", y);
    return 0;
}
